jsonrpc/eth_endpoint_test.go

```diff
@@ -242,6 +242,61 @@ func TestEth_GetNextNonce(t *testing.T) {
 	}
 }
 
+func TestEth_TxnType(t *testing.T) {
```

Comment:
Looking at this again, this doesn't actually make any requests and just validates decoding